drivers/united.js

```diff
@@ -0,0 +1,31 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class United extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const tickers = await request('https://unitedexchange.io/api-ticker');
+
+    return tickers.map((ticker) => new Ticker({
+      base: ticker.base_currency,
+      quote: ticker.target_currency,
+      high: parseToFloat(ticker.high),
+      low: parseToFloat(ticker.low),
+      close: parseToFloat(ticker.last_price),
+      bid: parseToFloat(ticker.bid),
+      ask: parseToFloat(ticker.ask),
+      baseVolume: parseToFloat(ticker.base_volume),
```

Comment:
if I put quoteVolume to be ticker.target_volume, test fails. can you please check that @wouthoekstra is test runs correctly ? because, both base and quote volume seems fine

Reply:
The issue is with the DASH_BTC market:               
 "base_volume": 2176.2663218099997,
 "target_volume": 3847.1639141000087